src/main/java/org/simplify4u/plugins/ArtifactResolver.java

```diff
@@ -437,7 +439,10 @@ public Configuration(SkipFilter dependencyFilter, SkipFilter pluginFilter, boole
             this.dependencyFilter = requireNonNull(dependencyFilter);
             this.pluginFilter = requireNonNull(pluginFilter);
             this.verifyPomFiles = verifyPomFiles;
-            this.verifyPlugins = verifyPlugins;
+            if (!verifyPlugins && verifyPluginDependencies) {
+                LOG.warn("Enabling configuration parameter 'verifyPlugins' as 'verifyPluginDependencies' is enabled.");
```

Comment:
Warning is not needed here.
My idea was to allow minimize number of options for user configuration.